order/views.py

```diff
@@ -0,0 +1,88 @@
+import json
+import uuid
+
+from django.http      import JsonResponse
+from django.views     import View
+
+from user.models      import User
+from product.models   import Product, Image
+from .models          import Order, OrderProduct, OrderStatus
+from utils.decorator  import authorization
+
+class OrderProductView(View):
```

Comment:
view 이름도 장바구니랑만 관련된 view니까 아래와 같이 수정하면 좋을 것 같습니다.
```suggestion
class CartView(View):
```

Reply:
네! 카트 확인